### PR TITLE
commit for first release

### PR DESCRIPTION
**Pull request by kriii**
This branch is for the first release of `ain-py`.

After these codes are passed review, I'll be doing follows.

1. Separate `utils.py` from `ain-py` to make `ain-util-py`.
2. Release `ain-util-py`.
3. Correct the `ain-py`.
4. Release `ain-py`.
5. Complete merge.

### CODE REVIEW

File: ain/ain.py
Comment (platfowner) [inline ain/ain.py:33]: minor: could you remove space between ')' and ':'? and period '.' at the end?

File: ain/utils.py
Comment (platfowner) [inline ain/utils.py:266]: minor: could you add period '.' and the end?

**Comment by kriii** [comment]
Fixed TODOs. PTAL

File: ain/ain.py
Comment (liayoo) [inline ain/ain.py:143]: could you leave TODOs for the functions that haven't been implemented?

**Comment by liayoo** [comment]
As a next step (after the 1-5), when you get a chance, could you add simple tests & describe how to run them in README.md?

**Comment by kriii** [comment]
> As a next step (after the 1-5), when you get a chance, could you add simple tests & describe how to run them in README.md?

Sure!

File: ain/ain.py
Comment (kriii) [reply ain/ain.py:144]: Added. PTAL

File: ain/ain.py
Comment (platfowner) [inline ain/ain.py:139]: could you add a period at the end of all todos as well?

**Review by platfowner — COMMENTED** [review]
LGTM with some minor comments.
Please wait for @liayoo 's approval.
Thanks!

**Review by liayoo — APPROVED** [review]
LGTM, thanks for the nice implementation!

**Comment by kriii** [comment]
Merged! Thanks for the review!

**Pull request merged**
